pages/activity/[name]/manage/administrator.tsx

```diff
@@ -12,9 +13,8 @@ import { AdministratorModal } from '../../../../components/User/ActivityAdminist
 import { HackathonAdminList } from '../../../../components/User/HackathonAdminList';
 import activityStore from '../../../../models/Activity';
 import { i18n } from '../../../../models/Translation';
-import { withRoute } from '../../../api/core';
 
-export const getServerSideProps = withRoute<{ name: string }>();
+export const getServerSideProps = compose(cache(), router<{ name: string }>);
```

Comment:
```suggestion
import { RouterProps } from 'next-ssr-middleware';

export const getServerSideProps = compose<{ name: string }, RouterProps<{ name: string }>>(router);
```

只有需要 SSR 的页面才需要缓存，登录页面（特别是管理页面）都不要。